### ISSUE TITLE
COMO CONCERTAR O VIDEO DOWLOADHELPER

### ISSUE THREAD

**Issue by STEFANYMIRANDA0** [volta-cli/volta#1855]
ESTAVA FUNCIONANDO PERFEITAMENTE. MAIS AGORA TA PEDINDO TIPO UMA ATUALIZAÇÃO, BAIXAR OUTRO APP

**Comment by charlespierce** [comment]
@STEFANYMIRANDA0 I don’t think this is connected to Volta. We don’t have anything to do with individual packages, you’ll need to reach out to the authors of the tool involved.
